### PR TITLE
Load more notifications | fix #1274

### LINKED ISSUE #1274: Add "Load More" In notifications
In notification add the option to "Load More"

### PR DESCRIPTION
**Pull request by developedBySJ**
## Changes
- Added Load More Button in the Notifications panel
- fixed #1274

loadMoreNotifications

### CODE REVIEW

File: src/Components/Notifications/NotificationsList.tsx [hunk 1]
```diff
@@ -90,27 +92,32 @@ export default function ResultList() {
...
-      <tr className="bg-white">
-        <td colSpan={4}>
-          <Loading />
-        </td>
-      </tr>
-    );
+  if (!offset && isLoading) {
+    <div className="flex items-center justify-center">
+      <CircularProgress color="primary" />
+    </div>;
```
Comment (ghost) [inline src/Components/Notifications/NotificationsList.tsx]: Is this component getting returned or intend to assign to a variable?
Typescript doesn't have feature to return anything without `return` keyword i guess 😅

File: src/Components/Notifications/NotificationsList.tsx [hunk 2]
```diff
@@ -90,27 +92,32 @@ export default function ResultList() {
...
-              defaultPerPage={RESULT_LIMIT}
-              data={{ totalCount }}
-              onChange={handlePagination}
-            /> */}
+        {isLoading && (
+          <div className="flex items-center justify-center">
+            <CircularProgress color="primary" />
+          </div>
+        )}
+        {totalCount > RESULT_LIMIT && offset < totalCount - RESULT_LIMIT && (
```
Comment (ghost) [inline src/Components/Notifications/NotificationsList.tsx:114]: Can you check if this condition will allow user to see last `(totalCount-offset)` notifications 
when `RESULT_LIMIT >= (totalCount-offset) > 0`.

I think setting offset to `min(RESULT_LIMIT, totalCount-offset) > 0` should work fine.

I think i have missed something. 😅 Let's discuss it together 🙂

**Review by abhinandanarya06 — CHANGES_REQUESTED** [review]
Fix the doubts given below.
Remaining things looks good to me. 👍🏻 
Just one problem i found not spefic to this PR that after some scrolling over notifications, reload and close button get disappears. Let's create the corresponding issue or you can solve it in this PR itself.

File: src/Components/Notifications/NotificationsList.tsx (hunk 2 above)
Comment (developedBySJ) [reply src/Components/Notifications/NotificationsList.tsx:114]: Suppose, `totalCount = 28`
Initially, latest 15 notifications will be loaded.
`offset = 0`
`offset < totalCount - RESULT_LIMIT`  conditions is `true` as `0 < 28 - 15`, so `Load More` button will appear

For second page, `offset = 15`
`offset < totalCount - RESULT_LIMIT`  conditions is `false` as `15 < 28 - 15`, so `Load More` button will not appear

File: src/Components/Notifications/NotificationsList.tsx (hunk 1 above)
Comment (developedBySJ) [reply src/Components/Notifications/NotificationsList.tsx:98]: My Bad, I forgot `return`😅

Comment (developedBySJ) [reply src/Components/Notifications/NotificationsList.tsx]: Thanks 😃,
I forgot to assign it to the variable `manageResults`😅.

File: src/Components/Notifications/NotificationsList.tsx (hunk 2 above)
Comment (ghost) [reply src/Components/Notifications/NotificationsList.tsx:114]: oh got it. missed what `offset` means 😅. 😂

**Review by abhinandanarya06 — APPROVED** [review]
LGTM. Good job!

**Comment by amruthavijay88** [comment]
Tested, 'load more' option works in notification bar.

**Pull request merged**